ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -589,6 +589,65 @@ sflow hardware acceleration module Linecard1
 sflow hardware acceleration module Linecard2
 no sflow hardware acceleration module Linecard3
 !
+snmp-server ipv4 access-list SNMP-MGMT vrf MGMT
+snmp-server ipv4 access-list onur
+snmp-server ipv6 access-list SNMP-MGMT vrf MGMT
+snmp-server ipv6 access-list onur_v6
+snmp-server engineID local 424242424242424242
+snmp-server contact DC1_OPS
+snmp-server location DC1
+snmp-server local-interface Loopback0
+snmp-server vrf Tenant_A_APP_Zone local-interface Loopback12
+snmp-server vrf MGMT local-interface Management1
+snmp-server view VW-READ iso included
+snmp-server view VW-WRITE iso included
+snmp-server community SNMP-COMMUNITY-1 ro onur
+snmp-server community SNMP-COMMUNITY-2 view VW-READ rw ipv6 SNMP-MGMT SNMP-MGMT
+snmp-server community SNMP-COMMUNITY-3 ro
+snmp-server group GRP-READ-ONLY v3 priv read v3read
+snmp-server group GRP-READ-WRITE v3 auth read v3read write v3write
+snmp-server user REMOTE-USER-IP-LOCALIZED GRP-REMOTE remote 42.42.42.42 v3 localized DEADBEEFCAFE123456 auth sha ShouldBeEncryptedPassword
+snmp-server user REMOTE-USER-IP-ONLY GRP-REMOTE remote 42.42.42.42 v3
+snmp-server user REMOTE-USER-IP-PORT GRP-REMOTE remote 42.42.42.42 udp-port 666 v3
```

Comment:
same as above

Reply:
I think this commands required some remote device.